### ISSUE TITLE
Replace quantitative distance description with a qualitative ones

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#244]
Once the distance scale for describing qualitative distance between the hands is updated in #242, I would like to use qualitative distance in all places, and no longer use quantitative distance, ie, distance described as "tick marks apart" when the Tick Marks or Numbered tick marks is checked.

If I am not mistaken, I think this "quantitative distance phrase" is currently only used in the Both Hands interaction. For example:
- hands 4 tick marks apart from each other, left on tick mark 4, right on tick mark 8
- hands just over4 tick marks apart from each other, left on tick mark 4, right around tick mark 8

Instead use qualitative distance:
- hands far from each other, left on tick mark 4, right on tick mark 8
- hands far from each other, left on tick mark 4, right around tick mark 8

This change will greatly reduce and simplify the responses.

**Note** that the region given above as an example might not be "far from" once the scale is updated in issue #242.

Also, note that at the last design meeting we discussed flipping the design so the object response describes proximity to ratio and the context response describes distance changes and hand position changes. I'm still sorting that issue out. It should be posted soon.

I think you said you would make all other changes first, but i still thought it was worth mentioning that a "flip the design" issue was coming.

**Comment by zepumph** [comment]
Sounds good to me! Please review.

**Comment by terracoda** [comment]
Looks like all the quantitative distance phrases have been replaced with qualitative distance phrases.
- slider context responses
- both hands object responses

**Comment by terracoda** [comment]
I think we can close this issue :-)

**Comment by terracoda** [comment]
And in the lock ratio responses, too.
